test/common/vmmanage.py

```diff
@@ -46,6 +46,10 @@ def run_install_script(machine, do_build, do_install, skips, arg, args):
     if install and args["containers"]:
         machine.execute("/var/lib/testvm/containers.install")
 
+    # TMP HACK FOR SELINUX
```

Comment:
So what I would suggest temporarily ... is executing this command during ```remotectl ensure``` or as a command in the service startup file?